hieradata/site/ls/role/icinga-master.yaml

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
Because of the parameters that resides in the private repo: the icinga-master domain, ldap credentials, site and zone (both pointing to ls), and the idea is to have one master per site with intercommunication between them.

Reply:
I'm not following your reasoning. The keys in this file will be identical between sites, correct?  If so, it should be moved to `hieradata/org/lsst/role/icinga-master.yaml`.

Reply:
Yeap, it could be done. The only parameter that wouldn't agree is the ldap server IP (which currently is pointing to the base)